src/rl/models/ppo_model.py

```
@@ -0,0 +1,189 @@
+from .actor import AutoregressiveActor
+from .value_net import ValueNet
+
+
+class LayoutOptimizationModel(nn.Module):
+    def __init__(
+        self,
+        num_categories: int,
+        embedding_dim: int,
+        numerical_feat_dim: int,
+        numerical_hidden_dim: Optional[int] = None,
+        gnn_hidden_dims: List[int] = [128, 128],
```

Ruff (0.14.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)